### ISSUE TITLE
Question about biometric authentication

### ISSUE THREAD

**Issue by AdaltonLeite** [w3c/payment-request#1039]
Hi Team, 

I have one more question about a feature in Payment Request. We're using Payment Handler to make a custom wallet. In this wallet, I need to authenticate myself to be able to list the payment methods, one of the steps is biometric authentication which requires access to the browser camera. Currently, outside of Payment Request, this application works, but in Payment Request I never get the permission modal for the browser to access my camera. 

Are we planning to have this kind of feature where we ask for permission to access the camera from the Payment Request? Or do we have any limitations on this?

**Comment by AdaltonLeite** [comment]
Hi @rsolomakhin, thank you for your reply. 

I'll formulate better what we're looking for and send you an email with more details. 

If I understand correctly, we don't currently have this functionality, right? It's more about mapping it from our side here in the team. 

We're working on this in parallel to see if it's feasible to have this functionality in Chrome.

**Comment by rsolomakhin** [comment]
> If I understand correctly, we don't currently have this functionality, right?

Yes, the [Web Payment Handler](https://w3c.github.io/payment-handler/) implementation in Chrome currently has disabled certain functionality, such as browser camera.
